sdk/trace/span_exporter.go

```diff
@@ -45,3 +48,14 @@ type SpanExporter interface {
 	// DO NOT CHANGE: any modification will not be backwards compatible and
 	// must never be done outside of a new major release.
 }
+
+type PartialExportError struct {
```

Comment:
How is this different from having the error in `otlp` or `otlptrace`?  Yes other exporters can theoretically create this error, but no other exporter has this kind of error.

Reply:
Discussed in SIG meeting, but for others interested: There are likely other exporters that support partial successes. For instance, [there is the sumologic exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/5a43b119f6c0faedf55e40d365bb73dd2e92cb6a/exporter/sumologicexporter/exporter.go#L294-L298) in the collector.